db/migrate/20190613045818_create_shares.rb

```diff
@@ -0,0 +1,11 @@
+class CreateShares < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_shares do |t|
+      t.references :user, null: false
+      t.references :birth_record, null: false
+      t.references :recipient, polymorphic: true, index: true, null: false
```

Comment:
i think you can track yeahs and nahs with `created_at`, and then `revoked_at`. Does that sounds like the plan?

Reply:
Why does it need to be polymorphic?
could it just be a regular foreign key to organisation_id ?

Reply:
the wireframes include the option to send to an arbitrary email address, so I'm planning ahead for having different kinds of model as the recipient